configuration/wireless/access-point-bridged.md

```diff
@@ -0,0 +1,163 @@
+
+# Setting up a Raspberry Pi as a bridged wireless access point
+
+The Raspberry Pi can be used as a bridged wireless access point within an existing Ethernet network. This will extend the network to wireless computers and devices.
+
+If you wish to create a standalone wireless network, consider instead setting up a [routed access point.](./access-point-routed.md)
+
+```
+                                         +- RPi -------+
+                                     +---+ Bridge      |
+                                     |   | WiFi AP     +-)))
+                                     |   | 192.168.1.2 |         +- Laptop ----+
+                                     |   +-------------+     (((-+ WiFi STA    |
+                 +- Router ----+     |                           | 192.168.1.5 |
+                 | Firewall    |     |   +- PC#2 ------+         +-------------+
+(Internet)---WAN-+ DHCP server +-LAN-+---+ 192.168.1.3 |
+                 | 192.168.1.1 |     |   +-------------+
+                 +-------------+     |
+                                     |   +- PC#1 ------+
+                                     +---+ 192.168.1.4 |
+                                         +-------------+
+```
+
+A bridged wireless access point can be created using the inbuilt wireless features of the Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero W, or by using a suitable USB wireless dongle that supports access point mode.
```

Comment:
Product names are usually expanded out to the actual product, i.e. Raspberry Pi 4B, Raspberry Pi 3B and Raspberry Pi 3B+. Also, Raspberry Pi Zero WH needs to be added.

Reply:
That actually needs to be expanded to the actual product names, since the Raspberry Pi 3A+ does not have a wireless interface.

Reply:
IIRC in all the documentation I've seen, "Raspberry Pi Zero W" is assumed to include "Raspberry Pi Zero WH"?

Also, is Raspberry Pi Zero W even a sensible suggestion for this, since although it has a built-in wireless interface, it doesn't have a built-in ethernet interface?

Reply:
> since the Raspberry Pi 3A+ does not have a wireless interface.

Err, yes it does :wink: https://www.raspberrypi.org/products/raspberry-pi-3-model-a-plus/

Reply:
> > since the Raspberry Pi 3A+ does not have a wireless interface.
> 
> Err, yes it does 😉 https://www.raspberrypi.org/products/raspberry-pi-3-model-a-plus/

You're quite right. I must have been thinking of the original A - sorry!